docker-compose.yml

```diff
@@ -100,6 +100,9 @@ services:
       - AB2D_KEYSTORE_LOCATION=${AB2D_KEYSTORE_LOCATION}
       - AB2D_KEYSTORE_PASSWORD=${AB2D_KEYSTORE_PASSWORD}
       - AB2D_KEY_ALIAS=${AB2D_KEY_ALIAS}
+      # Update coverage frequently so that tests aren't slowed down too much
```

Comment:
We run in production with defaults. Meaning that we pull new metadata jobs to run only every thirty seconds or so. To me this is fine because metadata jobs take awhile and this won't impact an EOB search very much in the grand scheme of things. It also prevents one worker from claiming all of the metadata jobs by chance.